deploy/assisted-installer-controller/assisted-installer-controller-pod.yaml.template

```diff
@@ -13,6 +13,8 @@ spec:
         - name: assisted-installer-controller
           image: {{.ControllerImage}}
           imagePullPolicy: Always
+          securityContext:
+            privileged: true
```

Comment:
What necessitates this?

Reply:
I am placing the cert under the standard /etc/pki/ca-trust/source/anchors/ directory on the host. Looks like hostpathing /etc/pki directory needs privileged access. 
Since we are not running `update-ca-certificates` command, it actually doesn't have to be /etc/pki. So may be I can place the cert under /etc/assisted-service. That would remove the need for privileged access.
And this is running as a job in the newly created OpenShift cluster. Is hostPath available in all OpenShift clusters?

Reply:
Changed the cert path on the node to /etc/assisted-service/service-ca-cert.crt and removed the privileged access.